devenv.nix

```
@@ -216,6 +216,16 @@ in {
       ];
     };
 
+    additionalMysqldSettings = lib.mkOption {
+      type = lib.types.attrsOf lib.types.str;
+      default = { };
+      description = "Additional mysqld settings";
+      example = {
+         "lower_case_table_names" = "1";
+         "sql_mode" = "STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION";
```

The quotation marks are not necessary, I personally would like to remove them.

```suggestion
         lower_case_table_names = "1";
         sql_mode = "STRICT_TRANS_TABLES,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION";
```